config/locales/client.en.yml

```diff
@@ -4803,6 +4803,9 @@ en:
       panels:
         forum:
           label: Forum
+      back_to_forum: "Back to Forum"
```

Comment:
Changing the locale keys is not ideal because it will invalidate the translations but it feels wrong to leave them as `admin.*`